CMakeLists.txt

```diff
@@ -33,6 +33,8 @@ if(IS_TESTING)
   endif()
 endif()
 
+# find MPI
+find_package(MPI REQUIRED)
```

Comment:
PUMIPic requires MPI 3 (it uses nonblocking collectives, specifically `MPI_Ialltoall`).  Can you test with the following on your system:

```
set(pumipic_USE_MPI_DEFAULT ON)                                                                                                                                                                                                                                                 
bob_public_dep(MPI 3) 
```

That will set MPI 3 as a transitive dependency of PUMIPic instead of implicitly relying on Omega_h to require it.